packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/Label.tsx

```diff
@@ -35,3 +35,11 @@ export const Label = ({ label }: LabelProps) => {
     </div>
   );
 };
+
+export const RawLabel = ({ label }: LabelProps) => {
+  return (
+    <div data-id={label.id} style={label.style}>
+      {label.text}
+    </div>
+  );
+};
```

Comment:
Let's use the real components instead of my flawed idea of using such "raw" versions

Reply:
Done